odoo/addons/base/views/res_users_views.xml

```diff
@@ -38,6 +38,7 @@
             <field name="arch" type="xml">
                 <!-- the user list is editable, but one cannot add or delete rows -->
                 <tree string="Users" editable="bottom" create="false" delete="false">
+                    <field name="user_id" column_invisible="True"/>
```

Comment:
Thanks for the pr, you need to add an xml comment next to the field to explain why it is necessary in the view.

Reply:
Can you add a test ?

Reply:
Will do both :raised_hands:

Reply:
I am trying to add test, to test Form behavior 
```
        user_internal = self.env['res.users'].create({
            'name': 'Internal',
            'login': 'user_internal',
            'password': 'password',
            'groups_id': [self.env.ref('base.group_user').id],
        })

        form = Form(self.env['change.password.wizard'].with_context(active_model='res.users', active_ids=user_internal.ids), view='base.change_password_wizard_view')
```
And I get `NotImplementedError: onchange() is implemented in module 'web`... which I don't know what to do about :thinking:

Reply:
The test can go to web in this case. Logically this implies that the invisible field should be declared in web but I don't think this improve clarity.
Your opinion @ryv-odoo  ?

Reply:
@Gorash  is it possible to merge this (because now we can't change password in master and that does disturb people who test dbs).. and then we figure out what test should look like? Because seems like the later might take time :smile:

Reply:
@Gorash , I think it is ok to test base's Forms in /web without declaring fields in web (seems very complicated only to be 100 % pure).